pkg/types/types.go

```diff
@@ -25,8 +25,8 @@ const (
 	ServerNetProto              = "tcp"
 	InfraAgentAddr              = "127.0.0.1"
 	InfraAgentPort              = "50001"
-	InfraManagerAddr            = "127.0.0.1"
-	InfraManagerPort            = "50002"
+	DefaultInfraManagerAddr     = "127.0.0.1"
```

Comment:
Is this the default or the one being set in pkg/inframanager/config/config.go above?

Reply:
This is the default. The user can specify a different one via config.yaml file.

Reply:
So, can we remove the below line from config.go:
viper.SetDefault("InfraManager.Addr", "localhost:50002")